eng/testing/performance/android_scenarios.proj

```diff
@@ -21,19 +25,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <HelixWorkItem Include="SOD - Android HelloWorld APK Size">
+    <HelixWorkItem Include="SOD - Android HelloWorld APK Size $(RunConfigsString)">
```

Comment:
Just a comment on including the RunConfigsString in the workitem name. It is not completely necessary to have the config in the name as the config data should be saved in the RunConfiguration information of the run. There is the benefit of being able to directly search for it in the name, but if you think that we will want to be able to slice only by the name before the config string in any views, then it may be worth keeping it out of the name and doing any name changes where we surface the data.

This is something we do for the other mobile runs though, so including it here is also alright.

Reply:
I agree and I removed it. We can see the configurations also in the job name so it is unnecessary to include it here.